app/src/main/java/com/github/lookupgroup27/lookup/model/map/Renderer.kt

```diff
@@ -17,50 +19,93 @@ class Renderer(private val context: Context) : GLSurfaceView.Renderer {
 
   private lateinit var textureManager: TextureManager
   private lateinit var skyBox: SkyBox
+  private lateinit var planet: Planet
 
   private var skyBoxTextureHandle: Int = -1 // Handle for the skybox texture
 
   /** The camera used to draw the shapes on the screen. */
   val camera = Camera()
-
+  // Temporary storage for the MVP matrix
+  private val mvpMatrix = FloatArray(16)
+  private val viewMatrix = FloatArray(16)
+  private val projectionMatrix = FloatArray(16)
+
```

Comment:
As Adrien said, all of the matrices matter is already handled in the Camera class. So this shouldn't be here.